### ISSUE TITLE
WCAG 2.1: Reflow - Text in the right pane in automated checks is truncated at 200% zoom

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#765]
**Describe the bug**
The following text in the right pane in automated checks is truncated
- Target app text
- Automated checks heading
- Failure message count sub text

**To Reproduce**
Adjust the following display settings in Windows :
- Display resolution to ```1280 * 1024```
- Scaling size to ```200%```

1. Run AI-Windows on any app that fails automated checks
2. Navigate to Tests -> Automated Checks
3. Observe the failures on the right pane

**Expected behavior**
Text content should be made available i.e. completely visible or be revealed via accessible controls

**Actual behavior**
Text content  is hidden

**Screenshots or .GIF**
![1](https://user-images.githubusercontent.com/22068434/79991603-59202c00-8467-11ea-956b-6807ec814e80.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise, Version1903
 - Accessibility Insights for Windows Version: Production Release: v1.1.1198.01

**Comment by adarshrema** [comment]
From internal testing for accessibility. To be combined with results from planned HC testing.
